pom.xml

```diff
@@ -86,7 +86,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>1.18.2</version>
+            <version>1.18.14</version>
```

Comment:
@AlexPerathoner  please consider 
https://github.com/projectlombok/lombok/issues/2599
https://github.com/projectlombok/lombok/issues/2607

1.18.14 has some breaking changes.

Reply:
After discussing with hudda10, we decided to keep version 1.18.14.
As I'm using this lombok version on quality service for a while now and we never add problems
The issues seemed related to eclipse so not relevant for our use case
If we see any lombok issue in the next days while testing we could try a higher version